### DISCUSSION TITLE
Does flux v2 remove resources applied with kubectl?

### DISCUSSION THREAD

**Discussion by karel0707** [fluxcd/flux2#1140] [Q&A] [answered]
The title explains my question. reconciliation deletes a resource that is removed in the git repo, but what with resources that are manually applied?

Thanks for your Time! :)

**Comment by stefanprodan** [comment, marked as answer]
Flux doesn't delete things that it doesn't manage, if it would do that then CoreDNS, CNI, the cluster autoscaler, etc would be wiped.

**Reply by karel0707** [reply]
thanks for the quick reply. now I get it. I am still new to flux and relativily new to kubernetes. I do an Internship about gitOps one team focuses around Flux and the other one ArgoCD.
